src/util/get-all-keys.js

```diff
@@ -1,10 +1,9 @@
 
-export default function keys(obj, enumerable = false) {
-  let listOfKeys = [
-    ...Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj),
-  ];
+export default function keys(obj, { enumerable = false } = {}) {
+  const method = enumerable ? 'getOwnPropertyNames' : 'keys';
+  const listOfKeys = Object[method](obj);
   if (typeof Object.getOwnPropertySymbols === 'function') {
-    listOfKeys = listOfKeys.concat(Object.getOwnPropertySymbols(obj));
+    listOfKeys.push(...Object.getOwnPropertySymbols(obj));
```

Comment:
nit: using concat is declarative and doing something like:

```js
return typeof Object.getOwnPropertySymbols === 'function' ?
  listofKeys.concat(Object.getOwnPropertySymbols(obj)) :
  listOfKeys;
```

would be a more functional way of doing it.